### ISSUE TITLE
[FEATURE] Implement localisation

### ISSUE THREAD

**Issue by frikandelworst** [BuildTheEarth/BuildTeamTools#30]
Since BTE is a worldwide project with people of a diverse background, it's important that these people can play in their own language if they so desire. Because of this, we should implement translations for all strings. If possible it would be preferred that these translations can be edited by build team staff members through an ingame GUI or something to make sure there are a lot of accurate translations.

**Comment by MineFact** [comment]
The library for this feature should be [LangLibs](https://github.com/Cinnazeyy/LangLibs), as this will become the standard translation system of BuildTheEarth.

**Comment by Zoriot** [comment]
Will there be a Crowdin section?
How BT specifc things will be handled?
Is it planned to translate Item names aswell, or what is the scope? @DonatiFilippo

**Comment by DonatiFilippo** [comment]
Yes, so there will be a Crowdin to handle this as soon as we pin down the final .yml with all the strings we need.
In general the localization system will be based on an enhanced version of the LangLibs, that is currently what I'm working on, and that will be integrated inside the plugin module, so that the paper plugin itself will not be a requirement.
Additionally a reworked version of the chat Util will be implemented. Also I would try to provide the opportunity to localise as much as possible, so yeah also item names 

For your question on "BT specific things" I'm not too sure of what you are referring to, can you provide an example?
